src/main/java/org/takes/rq/RqForm.java

```diff
@@ -221,8 +255,11 @@ public String single(final CharSequence name) throws IOException {
          * @param name Name of query param
          * @param def Default, if not found
          * @return Value of it
+         * @throws IOException if something fails reading parameters
          */
-        public String single(final CharSequence name, final String def) {
+        public String single(
+                final CharSequence name, final String def
```

Comment:
@lautarobock Same here, you can keep these params in the previous line and declare the throws clause in this line.